src/VisualStudio/Core/Def/TableDataSource/DiagnosticTableItem.cs

```diff
@@ -44,10 +44,10 @@ public override TableItem WithAggregatedData(string[] projectNames, Guid[] proje
             => Data.ProjectId;
 
         public override LinePosition GetOriginalPosition()
-            => new(Data.DataLocation.OriginalStartLine, Data.DataLocation.OriginalStartColumn);
+            => Data.DataLocation.OriginalFileSpan.StartLinePosition;
 
         public override string GetOriginalFilePath()
-            => Data.DataLocation.OriginalFilePath;
+            => Data.DataLocation.OriginalFileSpan.Path;
```

Comment:
genuinely terrified that anything in the diagnostic table is accessing the 'original' span, as opposed to the normalized, potentially mapped, span.  But not touching this for now.